driver/fillers_table.c

```diff
@@ -309,8 +309,14 @@ const struct ppm_event_entry g_ppm_events[PPM_EVENT_MAX] = {
 	[PPME_SYSCALL_USERFAULTFD_X] = {FILLER_REF(sys_autofill), 2, APT_REG, {{AF_ID_RETVAL}, {0} } },
 	[PPME_SYSCALL_OPENAT2_E] = {FILLER_REF(sys_empty)},
 	[PPME_SYSCALL_OPENAT2_X] = {FILLER_REF(sys_openat2_x)},
+	[PPME_SYSCALL_IO_URING_SETUP_E] = {FILLER_REF(sys_empty)},
```

Comment:
A good rule of thumb when rebasing is to avoid touching master; instead, append your code to the master one; in this case, everything should go after `PPME_SYSCALL_EXECVEAT_X`!

Reply:
Thanks for the info, will do!